install/kubernetes/cilium/templates/hubble-relay/configmap.yaml

```diff
@@ -29,15 +29,16 @@ data:
     sort-buffer-len-max: {{ .Values.hubble.relay.sortBufferLenMax }}
     sort-buffer-drain-timeout: {{ .Values.hubble.relay.sortBufferDrainTimeout }}
     {{- if .Values.hubble.tls.enabled }}
-    tls-client-cert-file: /var/lib/hubble-relay/tls/client.crt
-    tls-client-key-file: /var/lib/hubble-relay/tls/client.key
+    tls-hubble-client-cert-file: /var/lib/hubble-relay/tls/client.crt
+    tls-hubble-client-key-file: /var/lib/hubble-relay/tls/client.key
     tls-hubble-server-ca-files: /var/lib/hubble-relay/tls/hubble-server-ca.crt
     {{- else }}
     disable-client-tls: true
     {{- end }}
     {{- if and .Values.hubble.tls.enabled .Values.hubble.relay.tls.server.enabled }}
-    tls-server-cert-file: /var/lib/hubble-relay/tls/server.crt
-    tls-server-key-file: /var/lib/hubble-relay/tls/server.key
+    tls-relay-server-cert-file: /var/lib/hubble-relay/tls/server.crt
+    tls-relay-server-key-file: /var/lib/hubble-relay/tls/server.key
+    tls-relay-client-ca-files: /var/lib/hubble-relay/tls/hubble-server-ca.crt
```

Comment:
IIUC this means that we're forcing mTLS when `.Values.hubble.relay.tls.server.enabled`, in other words it's not possible after this patch to enable non-mutual TLS on Hubble Relay.

I think we need a new Helm value here to pick mTLS over non-mutual TLS, e.g. `.Values..hubble.relay.tls.server.mTLS=true`. Then maybe another one to specify a custom CA, but doesn't need to be part of this PR IMO.